File: rpc_client.py
import xmlrpc.client
from minhasClasses import Pessoa

if __name__ == "__main__":
    client = xmlrpc.client.ServerProxy("http://localhost:8000/rpc")
    pessoa = Pessoa("ze", 23)
    pessoa2 = client.adicionaAno(pessoa)
    pessoa2 = Pessoa(**pessoa2)
    print(pessoa2.idade)
    # print(client.somar(10, 15))

File: rb/rpc_client.rb
require 'xmlrpc/client'
require 'pp'
require './Pessoa.rb'

client = XMLRPC::Client.new('localhost', '/rpc' , 8000)

pessoa = Pessoa.new 'luiz', 23

# result = client.call2 'somar', 3,5
result = client.call2 'adicionaAno', pessoa.to_hash

puts result[1].values[1]
